grpc/service_config/service_config.proto

```diff
@@ -187,6 +187,8 @@ message GrpcLbConfig {
   // Multiple LB policies can be specified; clients will iterate through
   // the list in order and stop at the first policy that they support.
   repeated LoadBalancingConfig child_policy = 1;
+  // If specified, overrides the name of the target to be sent to the balancer.
+  optional string target_name = 2;
```

Comment:
This is proto3, so no need for "optional".

Reply:
Done.